doc/source/archiver.rst

```diff
@@ -0,0 +1,122 @@
+**********************************
+
+The Archiver Appliance is an EPICS logging system developed in
+a collaboration of SLAC, BNL, and MSU for collecting and
+storing measurements from various control devices. Data retrieval
+is based on the client-server interface using HTTP requests.
+Large-scale accelerators and experimental facilities may maintain
+multiple archivers. Within the databroker application, each archiver
+is represented by an ArchiverEventSource that needs to be configured
+and registred with ArchiverReader. ArchiverEventSource can either be
+integrated with databroker or used as a stand-alone class to address
+different requirements. The following sections highlight four major
+topics: ArchiverEventSource configuration, its integration with databroker,
+data retrival based on the databroker interface, and a stand-alone
+use case.
+
+Configuration
+-------------
+
+:class:`ArchiverEventSource` is configured via :class:`ArchiverReader`
+using a Python dictionary with four key-value pairs :
+
+* name: user-defined name of archiver
+* url: address of the Archiver Appliance Retrieval server
+* timezone: time zone (e.g., 'US/Eastern')
+* pvs: dictionary mapping user-defined names to EPICS PVs
+
+For example, the following code illustrates the configuration of
+two :class:`ArchiverEventSource` instances:
+
+.. code-block:: python
+
+    # CSX ArchiverEventSource
+    aes_config_csx = {
+		'name' : 'csx',
+		'url' : 'http://xf23id-ca.cs.nsls2.local:17668',
+		'timezone' : 'US/Eastern', 
+		'pvs' : {'pv1':'XF:23ID-ID{BPM}Val:PosXS-I'}}
+    
+    aes_csx = ArchiverReader.from_config(aes_config_csx)
+
+    # Accelerator ArchiverEventSource
+    aes_config_acc = {
+		'name' : 'accelerator',
+		'url' : 'http://arcapp01.cs.nsls2.local:17668', 
+                'timezone' : 'US/Eastern', 
+		'pvs' : {'test_watT' :'UT:SB1-Cu:1{}T:Prmry-I'}}
+
+    ar_acc = ArchiverReader.from_config(ar_config_acc)
+ 
+The configuration information is registered within :class:`ArchiverReader`.
+As a result, :class:`ArchiverEventSource` can
+be accessed later from :class:`ArchiverReader` using
+the :meth:`ArchiverReader.named` method as illustrated in the following code:
+
+.. code-block:: python
+
+   # CSX ArchiverEventSource
+   aes = ArchiverReader.named('csx')
+
+
+Integration with Databroker
+---------------------------
+
+.. currentmodule:: databroker
+
+The integration of ArchiverEventSource with :class:`Broker` uses the standard
+``add_event_source()`` method :
+
+.. code-block:: python
+
+   db = Broker.named('csx')
+
+   for name in ArchiverReader.archiver_names():
+	aes = ArchiverReader.named(name)
+	db.add_event_source(aes)
+
+   
+Data Retrieval 
+--------------
+
+After integrating ArchiverEventSources with :class:`Broker`, PV data can
+be retrieved with the standard :meth:`Broker.get_table` method:
+
+.. code-block:: python
+
+   # select header
+   hdr = db[69209]
+
+   stream_name = 'archiver_pv1'
+   df = db.get_table(hdr, stream_name) 	
```

Comment:
This can just be `db.table(stream_name)`

Reply:
@tacaswell, `db.table(stream_name)` or `hdr.table(stream_name)`?

Reply:
sorry! `hdr.table(stream_name)`